built-in-references/KubernetesService/container-allowed-images/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Ensure only allowed container images in AKS",
+    "policyType": "BuiltIn",
```

Comment:
>"policyType": "BuiltIn", [](start = 4, length = 24)

these aren't actually built-in policies right?  Users won't be able to create these policies with this property here

Reply:
> Whats the plan with the "limited-preview" part of the paths when this comes out of preview?  Everything will have to be duplicated for a while as built-ins with the new paths roll out

For public preview, the policies model is going to change from rego to YAML files. Hence placing the policies under limited-preview.

Reply:
Liz told me to provide the built-in definitions here and I referred the other built-in samples https://github.com/Azure/azure-policy/blob/master/samples/built-in-policy/enforce-tag-value/azurepolicy.json, they are also in similar fashion.
For creating new policies using these definitions, azurepolicy.parameters.json and azurepolicy.rules.json can be used with Azure PowerShell and CLI cmdlets.